### ISSUE TITLE
does fle work with insertMany or create 

### ISSUE THREAD

**Issue by mergingseas** [wheresvic/mongoose-field-encryption#78]
Hello,
Looks like there isn’t support for insertMany or create mongoose queries ?

**Comment by wheresvic** [comment]
Hi,

Yes this is a new feature and I will make a ticket for it. Would you like to help implement it?

**Comment by adidaslevy** [comment]
Hello!
Actually, I'm also interested with encryption of bulk operations.

I tried to run encryptFieldsSync in 'pre' hook of 'insertMany', but the document is still the model, and not actual document that going to DB. 

Anyway, because of only 'encrypt' method is exposed, it's the only thing I can use - I would prefer to run the encryptFields method, that does all for me. 

Also, I have created a wrapper plugin where it allows user to only state the fields he want to encrypt in schema itself in the form:


`export const MongoEncryptionSchema: Schema = new Schema({
  firstName: { type: String, required: true, encrypt: true },
  middleName: { type: String, required: true, encrypt: true },
  lastName: { type: String, required: true, encrypt: true },
  phoneNumber: { type: String, required: true, encrypt: true },
  email: { type: String, required: true, encrypt: true }
....
`
